### ISSUE TITLE
ModuleNotFoundError: No module named 'config'

### ISSUE THREAD

**Issue by olea** [wetneb/openrefine-wikibase#121]
Hi:

I can't run a docker instance in my Fedora 32 system. I follow the build instructions (just using podman instead of docker):

```
git clone https://github.com/wetneb/openrefine-wikibase
cd openrefine-wikibase/
cp config_docker.py config.py
podman-compose up
```

and everything builds but the openrefine-wikibase container can't run:

```
[...]
podman start -a openrefine-wikibase_reconcile_1
podman start -a openrefine-wikibase_redis_1
1:C 22 Aug 2021 17:41:08.310 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
1:C 22 Aug 2021 17:41:08.310 # Redis version=6.2.5, bits=64, commit=00000000, modified=0, pid=1, just started
1:C 22 Aug 2021 17:41:08.310 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
1:M 22 Aug 2021 17:41:08.311 * monotonic clock: POSIX clock_gettime
1:M 22 Aug 2021 17:41:08.313 * Running mode=standalone, port=6379.
1:M 22 Aug 2021 17:41:08.313 # Server initialized
1:M 22 Aug 2021 17:41:08.313 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
1:M 22 Aug 2021 17:41:08.314 * Ready to accept connections
Traceback (most recent call last):
  File "app.py", line 11, in <module>
    from wdreconcile.engine import ReconcileEngine
  File "/openrefine-wikibase/wdreconcile/engine.py", line 1, in <module>
    import config
ModuleNotFoundError: No module named 'config'
1
```

I've tried some python configuration experiment without success, the expected result as I really don't know.

What I'm doing wrong? 

Thanks

**Comment by wetneb** [comment]
Not knowing podman (and not knowing docker very well either) it's hard for me to imagine what could be going wrong here, sorry!

**Comment by thadguidry** [comment]
@olea It's easiest if you attach to the running docker container... and try things out manually to see where the issue might lie.
https://docs.docker.com/engine/reference/commandline/attach/
Do some Python commands within it to verify the Python environment and if `config.py` is available and more important...it's path and if it's accessible to the Python installation in the container.
`pip list`
or start a Python shell inside the container while attached and print the modules
```shell
$ python
Python 3.8 <r38:123456> 
>>> help('modules')
```

Basically your looking for that `config.py` that you copied to ensure it's readable and available to the container's Python runtime environment.

**Comment by wetneb** [comment]
Deployment issue within the scope of this repo.
